src/Data/Lens/Types.purs

```diff
@@ -27,21 +27,45 @@ import Data.Profunctor.Choice (class Choice)
 import Data.Profunctor.Closed (class Closed)
 import Data.Profunctor.Strong (class Strong)
 
--- | A general-purpose Data.Lens.
-type Optic p s t a b = p a b -> p s t
-type Optic' p s a = Optic p s s a a
+-- | Given a type whose "focus element" can always be retrieved,
+-- | a lens provides a convenient way to view, get, and transform
+-- | that element. 
+-- | 
+-- | `_2` is a Tuple-specific `Lens` available from `Data.Lens`, so:
+-- | ```purescript
+-- | > import Data.Lens
+-- | > over _2 String.length $ Tuple "ignore" "four"
+-- | (Tuple "ignore" 4)
+-- | ```
```

Comment:
I guess we should concoct an example that does take advantage of the `s t a b` form as well as `s a` here, since this is the general version.

Reply:
This example does, because it converts a string to an int. I added more text, though.